.spotbugs/spotbugs-exclude.xml

```diff
@@ -6,4 +6,10 @@
     <Match>
         <Bug pattern="EI_EXPOSE_REP2"/>
     </Match>
+    <Match>
+        <Bug pattern="DM_CONVERT_CASE"/>
+    </Match>
+    <Match>
+        <Bug pattern="ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD"/>
+    </Match>
```

Comment:
I guess you had to add this because of the usage of enableAutoCommit in the main (which is static) and an instance method. I would remove this and follow the comment I have later in this review.

Reply:
Done